packages/widget-chat/src/containers/chat-widget/index.js

```diff
@@ -123,14 +139,24 @@ export class ChatWidget extends Component {
 
   handleScroll() {
     const props = this.props;
+    const {
+      conversation,
+      spark,
+      widget
+    } = props;
+
+    props.setScrollPosition({scrollTop: this.activityList.getScrollTop()});
```

Comment:
Like this
`const { setScrollPosition } = props; setScrollPosition();`
instead of `props.setScrollPosition`